plugins/backstage-plugin-github-insights/src/components/InsightsPage/InsightsPage.test.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 RoadieHQ
```

Comment:
Since we had turned off the license check we hadn't been forced to uptake the new one